assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,189 @@
+
+#country-selector {
+  _-- {
+    color: inherit;
+  }
+  font-size: $font-size-sm;
+  margin-bottom: 24px;
+  text-align: start;
+  width: 100%;
+
+  a _-- {
+    color: inherit;
+  }
+
+  button:hover {
+    text-decoration: var(--link--hover--text-decoration, underline);
+  }
+
+  button:focus {
+    border-radius: var(--link--focus--border-radius, 4px);
+    outline: var(--link--focus--border, 2px solid transparentize($white, 0.6));
+    outline-offset: 0;
+  }
+
+  button:focus:not(:focus-visible) {
+    outline: 0;
+  }
+
+  .control-toggle {
+    align-self: flex-start;
+    background: transparent;
+    border: none;
+    color: var(--site-footer--color, $white);
+    cursor: pointer;
+    text-align: start;
+    white-space: nowrap;
+    width: 100%;
+
+    &::before,
+    &::after {
+      background-color: var(--site-footer--color, $white);
+      content: "";
+      display: inline-block;
+      width: 16px;
+    }
+
+    &::before {
+      height: 20px;
+      margin-inline-end: 8px;
+      mask: url("../../images/gp-g-logo.png") 0 0/16px 20px no-repeat;
+      vertical-align: middle;
+    }
+
+    &::after {
+      height: 12px;
+      margin-inline-start: 8px;
+      mask: url("../../images/chevron.svg") 0 0/16px 12px no-repeat;
+      transform: rotate(0.25turn);
+      transition: transform 300ms cubic-bezier(.86, 0, .07, 1);
+    }
+  }
+
+  .countries-list {
+    _-- {
+      background: var(--site-footer--background, $active-blue);
+    }
+    font-weight: 500;
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.5s cubic-bezier(0, 1.04, .38, .37) 0s;
+  }
+
+  .countries-list .container {
+    padding: 24px 24px $space-lg 16px;
+
+    @include medium-and-up {
+      padding: 24px 0 $space-lg;
+    }
+  }
+
+  &.open {
+    .country-selector-toggle-container {
+      border-bottom: 1px solid transparentize($white, 1);
+      transition-delay: 0s;
+    }
+
+    .control-toggle::after {
+      transform: rotate(-0.25turn);
+    }
+
+    .countries-list {
+      max-height: 5000px;
+      transition: max-height 1s linear 0s;
+    }
+  }
+
+  ul.countries {
```

Comment:
Can you remove this redundant `ul`, and the other redundant elements where a class is also specified in this file?

The problem with high CSS specificity is that you'll have to at least match the specificity in any overrides you do afterwards (and there are a few in this file). It results in more complex code that is prone to specificity-related regressions.

It also means that any child theme that wants to override CSS needs to also include a selector that is specific enough, and so will have the same issues.

We create a class for this specific purpose anyway, we don't intend to use it elsewhere. If `countries` by itself is not semantic enough, we can rename it.